packages/chrome-extension/popup.html

```
@@ -42,6 +42,14 @@ <h1>Audicle</h1>
     <div class="control-group">
       <button id="pause-resume-btn">一時停止</button>
     </div>
+    <div class="auth-group">
+      <div id="auth-status">Audicle 未ログイン</div>
+      <button id="audicle-login-btn">Audicle にログイン</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)